### DISCUSSION TITLE
Precompiled Windows64 OpenRadioss binaries are available, now!

### DISCUSSION THREAD

**Discussion by MarianBulla** [OpenRadioss/OpenRadioss#780] [Announcements]
Dear all, 
I am very pleased to announce that in addition to the precompiled OpenRadioss for Linux, we are now providing precompiled OpenRadioss binaries for native use on Microsoft Windows64 systems.

The zip file contains additional libraries that are needed with the OpenRadioss executives.

Happy modelling, and best regards,
Marian

**Comment by aru2244** [comment]
Hi can you Please put  documentation for How to Install, and Use OpenRadioss  in windows

**Comment by OlivierWienholtz** [comment]
Have a look on GitHub at the "How to Run OpenRadioss" section : https://github.com/OpenRadioss/OpenRadioss/blob/main/INSTALL.md
Windows is explained in.

**Comment by kv-memerson** [comment]
Hi! Is it possible to run implicit solver on the precompiled windows binaries?
